### DISCUSSION TITLE
How to disable MUC to push old messages?

### DISCUSSION THREAD

**Discussion by bromoapp** [processone/ejabberd#4005] [Questions] [answered]
Hi, I am creating a chat for in-game chat.
I don't want old messages or messages that were posted in MUC to be pushed to a new Occupant who just joined.
Imagine there are 1000 players who posted thousands of messages... then one player joins and is forced to accept thousands of messages.
How to disable this?

Thanks

**Comment by licaon-kter** [comment, marked as answer]
https://docs.ejabberd.im/admin/configuration/modules/#mod-muc ->
```
  mod_muc:
    access_mam: none
    history_size: 0
```
